os/linux/archlinux.go

```diff
@@ -0,0 +1,30 @@
+package linux
+
+import (
+	"strings"
+
+	"github.com/k0sproject/rig"
+	"github.com/k0sproject/rig/os"
+	"github.com/k0sproject/rig/os/registry"
+)
+
+// ArchlinuxARM provides OS support for ArchlinuxARM systems
+type ArchlinuxARM struct {
```

Comment:
Probably should just be ArchLinux as it will be detected and should work with non-arm arch as well.